spark.yaml

```diff
@@ -0,0 +1,47 @@
+package:
+  name: spark
+  version: 3.5.0
+  epoch: 0
+  description:
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - openjdk-17-jre
+      - bash
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - curl
+      - maven
+      - openjdk-17
+      - openjdk-17-default-jvm
+      - bash
+      - wolfi-base
+      - wolfi-baselayout
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/apache/spark
+      tag: v${{package.version}}
+      expected-commit: ce5ddad990373636e94071e7cef2f31021add07b
+
+  - runs: |
+      export LANG=en_US.UTF-8
+      export JAVA_HOME=/usr/lib/jvm/java-17-openjdk
+      cd /home/build/
+
+      mvn -DskipTests clean package
```

Comment:
./build/mvn -DskipTests clean package

Reply:
Even thought your in the build directory, it'd still use the system installed maven here, unless perhaps switching to ./mvn? 

When I was able to build locally I just passed in the full path from root